pkgs/misc/emulators/retroarch/cores.nix

```diff
@@ -811,8 +804,15 @@ in
     description = "Optimized port/rewrite of SNES9x 1.43 to Libretro";
     license = "Non-commercial";
     makefile = "Makefile";
+  };
+
+  snes9x2005-plus = mkLibRetroCore {
+    core = "snes9x2005-plus";
+    src = getCoreSrc "snes9x2005";
+    description = "Optimized port/rewrite of SNES9x 1.43 to Libretro, with Blargg's APU";
+    license = "Non-commercial";
+    makefile = "Makefile";
     makeFlags = [ "USE_BLARGG_APU=1" ];
-    postBuild = "mv snes9x2005_plus_libretro${stdenv.hostPlatform.extensions.sharedLibrary} snes9x2005_libretro${stdenv.hostPlatform.extensions.sharedLibrary}";
   };
```

Comment:
`snes9x2005` with `USE_BLARGG_APU` results on `snes9x2005-plus` (that was what we previously called `snes9x2005`). So this just fixes it, and add a proper `snes9x2005` (without the `USE_BLARGG_APU` set).